lib/ace/edit_session/bracket_match.js

```diff
@@ -37,6 +37,70 @@ var Range = require("../range").Range;
 
 function BracketMatch() {
 
+
+    this.getBracketsCoor = function(position) {
+        if (position.column == 0) return null;
+
+
+        var charBeforeCursor,
+            matchLeft,
+            charAfterCursor,
+            matchRight;
+        
+        var length = -1,
+            depthLeft = 0,
+            depthRight = 0;
+
+
+        while (!matchLeft && !matchRight) {
+
+            length++;
+
+            charBeforeCursor = this.getLine(position.row).charAt(position.column-length);
+            charAfterCursor = this.getLine(position.row).charAt(position.column+length);
```

Comment:
I suggest saving line to the local variable, because `getLine` might not be a cheap method